### PR TITLE
Replace warning with tiny-warning

### PR DESCRIPTION
**Pull request by TrySound**
This is the blocker for one more jss optimisation.

### CODE REVIEW

**Comment by TrySound** [comment]
300 bytes of parsed code
image

File: babel.config.js [hunk 1]
```diff
@@ -4,5 +4,5 @@ module.exports = {
     '@babel/react',
     '@babel/flow',
   ],
-  plugins: ['@babel/proposal-class-properties'],
+  plugins: ['@babel/proposal-class-properties', 'dev-expression'],
```
Comment (HenriBeck) [inline babel.config.js]: We should also maybe add the plugin to jss then to drop more kb

**Comment by TrySound** [comment]
@HenriBeck Could you release patch? I already added it locally but stuck with buggy warning package.

**Comment by HenriBeck** [comment]
yes, will do

**Pull request merged**